src/snowcli/plugins/sql/manager.py

```diff
@@ -29,4 +33,6 @@ def execute(
             query = file.read_text()
 
         single_statement = len(list(split_statements(StringIO(query)))) == 1
-        return single_statement, self._execute_string(query)
+        return single_statement, self._execute_string(
+            query, remove_comments=remove_comments
```

Comment:
Maybe keep our name `show_comments` until the very end and do here

```
remove_comments=not show_comments
```

WDYT? We will be consistency on our side